be/src/exec/tablet_sink_index_channel.cpp

```diff
@@ -758,6 +760,19 @@ Status NodeChannel::_wait_request(ReusableClosure<PTabletWriterAddBatchResult>*
                   << " commit " << _tablet_commit_infos.size() << " tablets: " << commit_tablet_id_list_str;
     }
 
+    if (closure->result.has_load_channel_profile()) {
+        const auto* buf = (const uint8_t*)(closure->result.load_channel_profile().data());
+        uint32_t len = closure->result.load_channel_profile().size();
+        TRuntimeProfileTree thrift_profile;
+        auto profile_st = deserialize_thrift_msg(buf, &len, TProtocolType::BINARY, &thrift_profile);
+        if (!profile_st.ok()) {
+            LOG(ERROR) << "Failed to deserialize LoadChannel profile, NodeChannel[" << _load_info << "] from ["
+                       << _node_info->host << ":" << _node_info->brpc_port << "], status: " << profile_st;
+        } else {
+            _runtime_state->load_channel_profile()->update(thrift_profile);
```

Comment:
1. one oltap sink operator can get profile from more than one channel, so latest profile of channel A can overwrite exsited profile of channel B?
2. if there is more than one olap table sink opeartor in one fragment, whose profile will be the final profile to FE?
3. different BE can have same channel's profile but in different time?
4. if one fragment's load profile is empty(doesn't get from any channel), what will it be hanlded

Reply:
1. The profile of Channel A will not overwrite channel B because they have different profile name. The profile name is like `Channel (host=127.0.0.1)`. A and B have different hosts.
2 and 3. The problem is similar because the profile for a channel is reported in multiple paths (multiple operators/BEs). To prevent the old profile overwrites the new one, I plan to add a version in `RuntimeProfile` to identify it's a new or old profile, and skip the update in `RuntimeProfile#update` if it's an old one
4. it will report an empty profile